Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.py

```diff
@@ -3874,8 +3871,180 @@ def get_detection_for_incident_command(incident_id: str) -> CommandResults:
+            else:
+                # All args should be a list. this is a fallback
+                url_filter = "{url_filter}{operator}{inp_arg}:'{arg_val}'".format(url_filter=url_filter, operator=op,
+                                                                                  inp_arg=key, arg_val=arg)
+    url_facet = '&facet=cve'
+    if argToBoolean(args.get('display_remediation_info')):
+        url_facet += "&facet=remediation"
+    if argToBoolean(args.get('display_evaluation_logic_info')):
+        url_facet += "&facet=evaluation_logic"
+    if argToBoolean(args.get('display_host_info')):
+        url_facet += "&facet=host_info"
+    # The url is hardcoded since facet is a parameter that can have serval values, therefore we can't use a dict
+    endpoint_url = '/spotlight/combined/vulnerabilities/v1?' + 'filter=' + \
+        url_filter + url_facet + "&limit=" + args.get('limit', '50')
+    vulnerability_response = http_request('GET', endpoint_url)
+    headers = ['CVE ID', 'CVE Severity', 'CVE Base Score', 'CVE Published Date', 'CVE Impact Score',
+               'CVE Exploitability Score', 'CVE Vector']
+    outputs = []
+    for vulnerability in vulnerability_response.get('resources'):
+        outputs.append({'CVE ID': vulnerability.get('cve', {}).get('id'),
+                        'CVE Severity': vulnerability.get('cve', {}).get('severity'),
+                        'CVE Base Score': vulnerability.get('cve', {}).get('base_score'),
+                        'CVE Published Date': vulnerability.get('cve', {}).get('published_date'),
+                        'CVE Impact Score': vulnerability.get('cve', {}).get('impact_score'),
+                        'CVE Exploitability Score': vulnerability.get('cve', {}).get('exploitability_score'),
+                        'CVE Vector': vulnerability.get('cve', {}).get('vector')
+                        })
+    human_readable = tableToMarkdown('List Vulnerabilities', outputs, removeNull=True, headers=headers)
+    return CommandResults(raw_response=vulnerability_response,
+                          readable_output=human_readable, outputs=vulnerability_response.get('resources'),
+                          outputs_prefix="CrowdStrike.Vulnerability", outputs_key_field="id")
+
+
+def cs_falcon_spotlight_list_host_by_vulnerability_command(args: dict) -> CommandResults:
+    """
+        Get a list of vulnerability by spotlight
+        : args: filter which include params or filter param.
+        : return: a list of vulnerabilities according to the user.
+    """
+    endpoint_url = '/spotlight/combined/vulnerabilities/v1'
+    if not args or not args.get('cve_ids'):
+        raise DemistoException('Please insert at least one cve_ids argument')
+    url_filter = 'cve.id:[\'' + "','".join(argToList(args.get('cve_ids'))) + '\']'
+    params = {'filter': url_filter, 'facet': 'host_info', 'limit': args.get('limit')}
+    vulnerability_response = http_request('GET', endpoint_url, params=params)
```

Comment:
Same as above. Split the command to request and command

Reply:
added a request function